api/src/OpenTelemetry/Logging/Core.hs

```diff
@@ -2,29 +2,43 @@
 {-# LANGUAGE DerivingStrategies #-}
 {-# LANGUAGE GeneralizedNewtypeDeriving #-}
 
-module OpenTelemetry.Logging.Core where
+module OpenTelemetry.Logging.Core (
+  LogRecord (..),
+) where
 
-import Data.Int (Int32, Int64)
+import Data.Int (Int64)
 import Data.Text (Text)
-import OpenTelemetry.Attributes (Attribute)
 import OpenTelemetry.Common
+import OpenTelemetry.LogAttributes (LogAttributes)
 import OpenTelemetry.Resource (MaterializedResources)
 import OpenTelemetry.Trace.Id (SpanId, TraceId)
 
 
-data Log body = Log
+{- | This is a data type that can represent logs from various sources: application log files, machine generated events, system logs, etc. [Specification outlined here.](https://opentelemetry.io/docs/specs/otel/logs/data-model/)
+Existing log formats can be unambiguously mapped to this data type. Reverse mapping from this data type is also possible to the extent that the target log format has equivalent capabilities.
+-}
+data LogRecord body = LogRecord
```

Comment:
Renamed because of [this](https://opentelemetry.io/docs/concepts/glossary/#log).